crates/ruff_dev/src/generate_options.rs

```diff
@@ -117,6 +117,21 @@ fn emit_field(output: &mut String, name: &str, field: &OptionField, parent_set:
         field.example
     ));
     output.push('\n');
+
+    if let Some(deprecated) = &field.deprecated {
+        output.push_str("> **Warning**\n");
+        output.push_str("> This option is deprecated");
```

Comment:
Can we remove this or otherwise modify the deprecation message? Right now, it reads "This option is deprecated: This option has been deprecated ..."

Reply:
I updated the messaging of the deprecated note and the formatting in the documentation generation. I hope you like this better. 
<img width="1231" alt="Screenshot 2023-10-19 at 09 32 38" src="https://github.com/astral-sh/ruff/assets/1203881/e4234485-47a1-4066-851f-182e5a131f43">